location/templates/location.html

```diff
@@ -121,26 +124,6 @@ <h4 class=" mb-0">
 </div>
 
 
-<h2>Apartments</h2>
-{% if zillow_list %}
-<ul>
-{% for apt in zillow_list %}
```

Comment:
Zillow information is optional to Apartment model. That's why I used `zillow_list` here instead of `object.apartment_set.all`. Until we have apartment detail page, could you modify https://github.com/gcivil-nyu-org/fall2019-cs-gy-6063-team-five_fries/pull/198/files#diff-bff5ccfb02795886ca79e6c5ac90dba8L47 to use `zillow_list`?